sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -197,6 +199,84 @@ public async Task UpdateCapabilitiesAsync()
             Assert.AreEqual(200, updateOperation.GetRawResponse().Status);
         }
 
+        [Test]
+        public async Task ReleaseUnathorizedNumber()
+        {
+             const string PhoneNumber = "+14255550123";
+            var client = CreateClient();
+            try
+            {
+                var releaseOperation = await client.StartReleasePhoneNumberAsync(PhoneNumber);
+            }catch (Exception e)
+            {
+                Assert.NotNull(e);
```

Comment:
Can we do better than just asserting that the exception is not null?
e.g. check for the message matching a string, etc

Reply:
The intention of this test case its to check the server throws an exception, In the case if for some reason people from the server side  changes the message this test cases will fail and at the end that is not something that should be wrong, the message is just informative and not something the SDK  needs to validate

Reply:
We should also be able to check the type of the exception and check that the message exists.